turbinia/api/cli/README.md

```diff
@@ -1,164 +0,0 @@
-# Turbinia API CLI tool (turbinia-client)
```

Comment:
On the first point, the pypi package should ideally have a descriptive README so that it is also rendered when you view it at pypi.org. If we move the documentation/README to the Turbinia docs directory, it won't be accessible to the PyPi package and we'd have to put some sort of placeholder that points to the Turbinia readthedocs page. It seems better to me to just keep the readme as part of the package and link to it from the main Turbinia docs.

On the second point, we already have both .rst and .md files in the docs (e.g. index.rst) for linking purposes. The main reason for having a rST file for the client's README is that it's linked to the main Turbinia docs via an ```:include:``` directive rather than a direct reference to a file within the same doc root directory like all the other files. :include: expects a rST file from what i can tell. I could not find a more elegant or simpler way to link to a .md file outside the doc root without having to hack conf.py somehow. I'd opt to use rST to include this README.rst so we only change it in one place and the same file is rendered in both Readthedocs and PyPi but open to changing this around.

Reply:
as a side note, i should mention converting from .md to .rst was pretty simple via the ```m2r``` pip package if we ever need to do that.

Reply:
Ok, it sounds fine to just keep in in place then.